### PR TITLE
Jakarta Authentication 3.0 Plan Review

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

## Plan Review PR template
When creating a specification project plan review, create PRs with the content defined as follows.

Include the following in the PR:

- [X]   A directory in the form wombat/x.y where x.y is the release major.minor version.
- [X]   An index page wombat/x.y/_index.md following template
- [X]   Title should be on the form "Jakarta Wombat X.Y (under development)"
- [X]   Includes a plan for the release. Usually, a couple of statements is enough. Otherwise, a link to a document describing the release.
- [X]   N/A - Includes a detailed description of any backward incompatibility (Mark with N/A and check if none)
- [X]   N/A - Declares optional features if there are any (Mark with N/A and check if none)
- [X]   Includes minimum Java SE version
- [X]   N/A - Describe any updates or required changes including splitting out the TCK (Mark with N/A and check if not planned)
- [X]   Link to updated release record
https://projects.eclipse.org/projects/ee4j.authentication/releases/3.0

### CODE REVIEW

File: authentication/3.0/_index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: "Jakarta Authentication 2.0 (under development)"
```
Comment (pzygielo) [inline authentication/3.0/_index.md:2]:
```suggestion
title: "Jakarta Authentication 3.0 (under development)"
```

Comment (arjantijms) [reply authentication/3.0/_index.md]: Thanks, well spot! :)

**Review by keilw — APPROVED** [review]

**Comment by keilw** [comment]
@kwsutter @arjantijms This one looks fine for me, should I merge it now or after the ballot?

**Comment by edbratt** [comment]
> @kwsutter @arjantijms This one looks fine to me, should I merge it now or after the ballot?

By custom, the merge is dependent on concluding a successful ballot. Go ahead and start the ballot, but don't merge the PR until the ballot successfully concludes.

**Comment by keilw** [comment]
Thanks, then I'll start the ballot for Authorization.

**Pull request merged**